ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -126,13 +138,45 @@ public void processPriorCommands() {
     ksqlEngine.getPersistentQueries().forEach(PersistentQueryMetadata::start);
   }
 
+  void fetchAndRunCommands() {
+    final List<QueuedCommand> commands = commandStore.getNewCommands(NEW_CMDS_TIMEOUT);
+    if (commands.isEmpty()) {
```

Comment:
nit: unnecessary check

Reply:
Well, it avoids an unnecessary trace message which, given the `getNewCommands` message has a timeout and doesn't just block indefinitely, has some benefit.  Though I admit, not much.